src/dhtproto/client/request/Remove.d

```diff
@@ -0,0 +1,97 @@
+/*******************************************************************************
+
+    Client DHT Remove request definitions / handler.
+
+    The Remove request attempts to remove one record from the specified DHT
+    channel. This works as follows:
+        1. The client selects a connected DHT node from its registry, based on
+           the key of the record to be read.
+        2. A request is sent to the selected node, asking for the specified
+           record in the specified channel to be removed.
+        3. The request ends when either the record is removed from the node, is
+           found to not exist, or the node could not handle the request due to
+           an error.
+
+    During a data redistribution, more than one node may be responsible for a
+    given key. In this case, the node which was most recently reported as being
+    responsible for the key is queried first, followed by others (in order)
+    until the record is removed, an error occurs, or no node has the record.
```

Comment:
This assumes that if, during the redistribution, the record exist on more than one node, it's enough to remove it just from the most recent one, as it will cease to exist on the nodes that were previously responsible for it?

Reply:
Oof, good point. This isn't correct. It needs to be removed from all nodes.

Reply:
Updated.